test/docfx.Test/DocfxTest.cs

```diff
@@ -103,6 +103,35 @@ public static void Run(TestData test, DocfxTestSpec spec)
             }
         }
 
+        private static bool IsOutputTypePageJson(DocfxTestSpec spec)
+        {
+            var existInInputs = IsExist(spec.Inputs);
+
+            var existInRepos = false;
+            foreach (var (_, repo) in spec.Repos)
+            {
+                foreach (var item in repo)
+                {
+                    existInRepos = existInRepos || IsExist(item.Files);
+                }
+            }
+
+            return existInInputs || existInRepos;
+        }
+
+        private static bool IsExist(IDictionary<string, string> dic, string expect = "outputType: pageJson")
```

Comment:
nit: consider move this method as an inner method inside `IsOutputTypePageJson`. Could also use _LINQ_ to simplify `IsExist` and `IsOutputTypePageJson`.